app/src/main/java/de/westnordost/streetcomplete/quests/OsmQuestAnswerListener.java

```diff
@@ -17,4 +18,7 @@
 
 	/** Called when the user chose to skip the quest */
 	void onSkippedQuest(long questId, QuestGroup group);
+
+	/** Called when the user wants to leave a note which is not related to a quest */
+	void onLeaveNote(String note, ArrayList<String> imagePaths, LatLon position);
```

Comment:
Hmm, as the callback doesn't have anything to do with quests (or answering them), the method should be in an own interface and not in this.